### PR TITLE
Remove ES matrix from CI

### PR DESCRIPTION
**Pull request by obulat**
## Fixes

This is a follow up to the removing Elasticsearch 7

## Description

This PR reverts the changes in #2832 that allowed us to test API and ingestion server on both ES versions.

## Testing Instructions

CI should pass.
After the CI passes and the PR is approved, the comments should be deleted.

## Checklist

- [ ] My pull request has a descriptive title (not a vague title like`Update index.md`).
- [ ] My pull request targets the _default_ branch of the repository (`main`) or a parent feature branch.
- [ ] My commit messages follow [best practices][best_practices].
- [ ] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [ ] I tried running the project locally and verified that there are no visible errors.
- [ ] I ran the DAG documentation generator (if applicable).

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Review by dhruvkb — APPROVED** [review]
Nice cleanup, thanks!

File: api/api/apps.py [hunk 1]
```diff
@@ -6,4 +6,4 @@ class ApiConfig(AppConfig):
     default_auto_field = "django.db.models.AutoField"
 
 
-# Comment to force CI
+# Comment to force CI. Don't forget to delete before merging.
```
Comment (krysal) [inline api/api/apps.py:9]: What does this mean? When should these comments (with the one in `ingestion_server/gunicorn.conf.py`) be removed?

**Review by krysal — COMMENTED** [review]
Despite a question, everything looks good to me.

File: api/api/apps.py (hunk 1 above)
Comment (sarayourfriend) [reply api/api/apps.py:9]: They can be removed now. Removing them will trigger the CI as well :slightly_smiling_face:

Comment (obulat) [inline api/api/apps.py:9]:
```suggestion
# Comment to force CI. Don't forget to delete before merging.
```

File: ingestion_server/gunicorn.conf.py [hunk 2]
```diff
@@ -43,4 +43,4 @@
 }
 loglevel = "debug"
 
-# Comment to force CI
+# Comment to force CI. Don't forget to delete before merging.
```
Comment (obulat) [inline ingestion_server/gunicorn.conf.py:46]:
```suggestion
```

File: ingestion_server/gunicorn.conf.py [hunk 3]
```diff
@@ -42,5 +42,3 @@
     },
 }
 loglevel = "debug"
```
Comment (obulat) [inline ingestion_server/gunicorn.conf.py:44]:
```suggestion
loglevel = "debug"

```

File: api/api/apps.py [hunk 4]
```diff
@@ -4,6 +4,3 @@
 class ApiConfig(AppConfig):
     name = "api"
     default_auto_field = "django.db.models.AutoField"
```
Comment (obulat) [inline api/api/apps.py:6]:
```suggestion
    default_auto_field = "django.db.models.AutoField"

```

File: api/api/apps.py (hunk 1 above)
Comment (obulat) [reply api/api/apps.py]: @krysal, I removed the comments since we've checked that the CI passes.
Now this PR is ready for re-review.

**Pull request merged**